tests/components/UserActions/UserActivity.spec.ts

```diff
@@ -201,20 +201,33 @@ describe('UserActivity', () => {
     });
 
     describe('search event', () => {
-        it('should display the "User Query" as event title when there is a query expression', () => {
-            const mock = Mock.advancedComponentSetup<UserActivity>(
-                UserActivity,
-                new Mock.AdvancedComponentSetupOptions(null, { userId: 'testuserId' }, env => {
-                    fakeUserProfileModel(env.root, sandbox).getActions.returns(Promise.resolve([FAKE_USER_SEARCH_EVENT]));
-                    return env;
-                })
-            );
-
-            return delay(() => {
-                const clickElement = mock.cmp.element.querySelector('.coveo-search');
-
-                expect(clickElement).not.toBeNull();
-                expect(clickElement.querySelector<HTMLElement>('.coveo-title').innerText).toBe('User Query');
+        ['omniboxAnalytics', 'userActionsSubmit', 'omniboxFromLink', 'searchboxAsYouType', 'searchboxSubmit', 'searchFromLink'].map(cause => {
```

Comment:
@louis-bompart is gonna love that, but I don't mind.

Reply:
I'm the one that asked for it :D